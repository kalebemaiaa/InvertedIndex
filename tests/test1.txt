QueryOr(QueryWord("presidente"), QueryWord("ministro"))
4585.raw
11497.raw
10854.raw
5341.raw
1879.raw
1654.raw
1492.raw
11710.raw
5304.raw
11430.raw
4862.raw
5335.raw
677.raw
3882.raw
11313.raw
422.raw
2966.raw
10905.raw
10891.raw
11364.raw
1459.raw
1561.raw
3583.raw
4305.raw
10866.raw
40.raw
1309.raw
5152.raw
4663.raw
5043.raw
388.raw
2948.raw
1471.raw
3167.raw
4210.raw
1900.raw
5852.raw
6295.raw
1046.raw
5473.raw
5826.raw
11860.raw
502.raw
2971.raw
2361.raw
19.raw
1183.raw
1433.raw
98.raw
5507.raw
2334.raw
11429.raw
11357.raw
2656.raw
11387.raw
25.raw
6133.raw
2389.raw
4319.raw
18.raw
5835.raw
10991.raw
5253.raw
3243.raw
3814.raw
11596.raw
874.raw
2645.raw
11251.raw
3932.raw
2012.raw
60.raw
4504.raw
443.raw
6414.raw
6239.raw
2519.raw
4120.raw
1261.raw
5067.raw
11183.raw
11008.raw
4748.raw
1526.raw
5579.raw
1986.raw
11259.raw
6191.raw
1563.raw
6274.raw
1142.raw
2657.raw
190.raw
3199.raw
4557.raw
6034.raw
2543.raw
11883.raw
2583.raw
2472.raw
1390.raw
4814.raw
1818.raw
5047.raw
885.raw
4912.raw
465.raw
11197.raw
436.raw
4551.raw
11949.raw
38.raw
525.raw
1653.raw
1189.raw
3264.raw
2652.raw
184.raw
8764.raw
4589.raw
4947.raw
3969.raw
1466.raw
4920.raw
2242.raw
61.raw
5543.raw
560.raw
11083.raw
1535.raw
5944.raw
10907.raw
582.raw
2178.raw
2812.raw
1823.raw
11159.raw
1400.raw
823.raw
8897.raw
4043.raw
6347.raw
5167.raw
2171.raw
1413.raw
4704.raw
3538.raw
2772.raw
6409.raw
835.raw
4610.raw
3239.raw
11735.raw
1801.raw
1667.raw
5367.raw
400.raw
3835.raw
1787.raw
5382.raw
4176.raw
3012.raw
718.raw
4168.raw
28.raw
2521.raw
1463.raw
11701.raw
3614.raw
3496.raw
11342.raw
5450.raw
11017.raw
6234.raw
11484.raw
11810.raw
4497.raw
11863.raw
5724.raw
4890.raw
2968.raw
4263.raw
11139.raw
2692.raw
2669.raw
426.raw
3307.raw
3762.raw
1701.raw
3941.raw
6231.raw
8893.raw
10753.raw
4889.raw
5188.raw
1152.raw
49.raw
641.raw
382.raw
5411.raw
