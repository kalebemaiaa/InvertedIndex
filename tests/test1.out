18
19
25
28
38
40
60
184
190
382
388
400
426
436
443
465
502
525
582
641
718
823
874
885
1046
1152
1189
1261
1309
1390
1400
1413
1433
1459
1463
1492
1563
1653
1654
1667
1701
1787
1801
1818
1823
1879
1900
2012
2171
2178
2242
2361
2389
2472
2519
2521
2645
2652
2656
2657
2669
2692
2812
2948
2966
2968
2971
3012
3167
3199
3239
3264
3307
3583
3614
3762
3814
3835
3882
3932
3941
3969
4043
4120
4168
4176
4263
4305
4504
4551
4557
4589
4663
4704
4748
4862
4889
4890
4912
4920
4947
5043
5047
5067
5152
5188
5253
5304
5335
5367
5382
5411
5450
5473
5543
5724
5826
5835
5852
5944
6034
6191
6231
6295
6347
8764
8893
10854
10891
10905
10907
10991
11008
11017
11083
11159
11183
11251
11313
11342
11357
11364
11429
11484
11497
11596
11701
11710
11810
11860
11863
11883
11949
